pages/linux/ddrescue.md

```diff
@@ -0,0 +1,12 @@
+# ddrescue
+
+> Data recovery tool that reads data from damaged block devices.
+> More information: <https://www.gnu.org/software/ddrescue/>.
+
+- Take an image of a device, creating a log file:
+
+`sudo ddrescue {{/dev/sdb}} {{path/to/image.dd}} {{path/to/ddrescue.log}}`
```

Comment:
path/to/log.txt here and after.

Reply:
@einverne - any reason you marked this as resolved ? It doesn't look resolved to me.

Reply:
Oh, sorry, I misunderstood. I was thinking the `{{path/to/ddrescue.log}}` is the modified version. You mean this, right ?

```suggestion
`sudo ddrescue {{/dev/sdb}} {{path/to/image.dd}} {{path/to/log.txt}}`
```

Reply:
Can I ask why you would rather it be a .txt instead of a .log?  IDC to change it, just my habits have always been to name log files *.log

Reply:
I don't have an issue with the extension. I would like to remove "ddrescue" from the file name since I feel it is redundant, and all file output placeholders that we have are just generic file names. I would be okay with "output.log" also.